### PR TITLE
refactored courses model, updated seed file and migration file to include instructor_name

### PR DESCRIPTION
**Pull request by theecolya**
## Description

courseModel.js was updated to include instructor_name in response data so that components using data from the courses endpoint had access to instructor_name.

Fixes # (issue)
No instructor name available in database

## Type of change

- [ ✓] New feature (non-breaking change which adds functionality)
- [ ✓] This change requires a documentation update

## Checklist:

- [ ✓] My code follows the style guidelines of this project
- [ ✓] I have performed a self-review of my own code
- [ ✓] I have removed unnecessary comments/console logs from my code
- [ ✓] I have made corresponding changes to the documentation if necessary (optional)
- [ ✓] My changes generate no new warnings
- [ ✓] I have checked my code and corrected any misspellings
- [ ✓] No duplicate code left within changed files
- [ ✓] Size of pull request kept to a minimum
- [ ✓] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by lisamdespain — APPROVED** [review]
This looks really good. Migration file (in profiles) sets up the table for instructor_name, seed file populates instructor_name, then getAllCourses in the model adds the field to the query. Great job implementing this functionality for the FE (and good thing we're not setting up instructors without a name!). Thanks!

**Review by Roadlyfe — COMMENTED** [review]
Code looks good and clean. nice work!

**Review by Roadlyfe — APPROVED** [review]
Code looks good and clean. nice work!

File: README.md [hunk 1]
```diff
@@ -90,6 +90,7 @@ avatarUr(not required): string,
 ```
 {
   instructor_id: INCREMENT (primary key, auto-increments, generated by database),
+  instructor_name: STRING (required),
```
Comment (macrowell70) [inline README.md:93]: Glad you've updated the documentation here. That could have been easy to overlook

**Review by macrowell70 — APPROVED** [review]
All looks good here. Nice catch adding to the Readme file when you made the video.

**Pull request merged**